### ISSUE TITLE
Tippecanoe-decode issues

### ISSUE THREAD

**Issue by joh10891** [developmentseed/label-maker#92]
After installing tippecanoe using 
`!git clone https://github.com/mapbox/tippecanoe.git 
!cd tippecanoe; make ,` 

when I try to run`!label-maker labels`, I get 

`FileNotFoundError: [Errno 2] No such file or directory: 'tippecanoe-decode': 'tippecanoe-decode'`

I checked, and tippecanoe appears in my directory. Is there a specific place that I need to put tippecanoe for it to be found by label-maker, or vice versa?

**Comment by drewbo** [comment]
@joh10891 any idea what the problem was (in case someone else runs into this)?

**Comment by joh10891** [comment]
Yes. I know it's not best practice and there's probably a better solution, but after installing tippecanoe, I moved everything from tippecanoe to bin using 
`from distutils.dir_util import copy_tree
copy_tree("/content/tippecanoe/", "/bin/")`
Which solved the probelms and label-maker was able to utilize it.

**Comment by mjvakili** [comment]
Hi @joh10891 I have faced the same issue. I already have tippecanoe and label-maker in my /bin/ directory. What did you exactly move to bin?
